js/src/components/paid-ads/ads-campaign.js

```diff
@@ -25,24 +25,29 @@ import FaqsSection from './faqs-section';
 /**
  * Renders the container of the form content for campaign management.
  *
- * Please note that this component relies on an CampaignAssetsForm's context and custom adapter,
- * so it expects a `CampaignAssetsForm` to existing in its parents.
+ * Please note that this component relies on a CampaignAssetsForm's context and custom adapter,
+ * so it expects a `CampaignAssetsForm` to exist in its parents.
  *
  * @fires gla_documentation_link_click with `{ context: 'create-ads' | 'edit-ads' | 'setup-ads', link_id: 'see-what-ads-look-like', href: 'https://support.google.com/google-ads/answer/6275294' }`
  *
  * @param {Object} props React props.
  * @param {Campaign} [props.campaign] Campaign data to be edited. If not provided, this component will show campaign creation UI.
  * @param {() => void} props.onContinue Callback called once continue button is clicked.
+ * @param {() => void} props.isLoading Callback called once continue button is clicked.
+ * @param {string} props.submitButtonText Text to display on submit button.
  * @param {'create-ads'|'edit-ads'|'setup-ads'} props.trackingContext A context indicating which page this component is used on. This will be the value of `context` in the track event properties.
  */
 export default function AdsCampaign( {
 	campaign,
 	onContinue,
+	isLoading,
+	submitButtonText,
 	trackingContext,
 } ) {
 	const isCreation = ! campaign;
 	const formContext = useAdaptiveFormContext();
 	const { isValidForm } = formContext;
+	const continueButtonText = submitButtonText || __( 'Continue', 'google-listings-and-ads' );  
```

Comment:
@kt-12 How about we have the default value set for the parameter here: https://github.com/woocommerce/google-listings-and-ads/pull/2577/files#diff-7679c27bf3a373cf135b63e0095de040d052c742d825f0896246656c0c0b53f0R44
instead of having a conditional here?

Reply:
Almost missed this one, Sure I'll update the defaults.

Reply:
Updated.